client-runtime/src/test/java/com/microsoft/rest/RestProxyWithRxNettyTests.java

```diff
@@ -2,14 +2,21 @@
 
 import com.microsoft.rest.http.ChannelHandlerConfig;
 import com.microsoft.rest.http.HttpClient;
-import com.microsoft.rest.http.RxNettyAdapter;
-import com.microsoft.rest.policy.RequestPolicy;
+import com.microsoft.rest.http.NettyAdapter;
+import com.microsoft.rest.policy.LoggingPolicy;
+import com.microsoft.rest.policy.RequestPolicy.Factory;
 
+import java.util.ArrayList;
 import java.util.Collections;
 
 public class RestProxyWithRxNettyTests extends RestProxyTests {
+    private static NettyAdapter adapter = new NettyAdapter(new ArrayList<Factory>() {
+        {
+            add(new LoggingPolicy.Factory(LogLevel.BODY_AND_HEADERS));
+        }}, Collections.<ChannelHandlerConfig>emptyList());
+
     @Override
     protected HttpClient createHttpClient() {
-        return new RxNettyAdapter(Collections.<RequestPolicy.Factory>emptyList(), Collections.<ChannelHandlerConfig>emptyList());
+        return adapter;
```

Comment:
Using a shared fixture can lead to tests that are very difficult to debug. Is there a reason you're opting to not create a new adapter for each test?

Reply:
Because the channel pool is shared between all the tests this way. Performance is improved. I should be able to create multiple clients yet still let them share the same channel pool.